### PR TITLE
Workout cards

### PR DESCRIPTION
**Pull request by Git-Hannah**
## Motivation

The exercise cards should be in a component
They are now, it is not done yet though

### CODE REVIEW

File: src/molecules/card-workout/index.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		duration: 600.0, // 10min
+	},
+	{
+		id: "Clown",
+		name: "Clown",
+		description:
+			"Scratch your nose to feel it, then imagine your nose has an eye which wants to see everything first. Explore your surroundings, leading with your nose!",
+		duration: "5min", // 5 minutes -> 5*60*1000
+	},
+];
```
Comment (doemser) [inline src/molecules/card-workout/index.jsx]: this could be an ion-file in your Atomic Design which you then import to this file

**Review by doemser — APPROVED** [review]
Good job!

File: src/molecules/card-workout/index.jsx (hunk 1 above)
Comment (Git-Hannah) [reply src/molecules/card-workout/index.jsx]: Great note! I was wondering where I should put it, once it is a .json!

Comment (doemser) [reply src/molecules/card-workout/index.jsx]: if this becomes a .json at some point, you would probably want to place it in an "api" folder i think.

Comment (Git-Hannah) [reply src/molecules/card-workout/index.jsx]: good point...

**Review by doemser — APPROVED** [review]
looks good, as far as I can tell

**Pull request merged**